SR2020/2019-10-11-kickstart-details-confirmation.md

```diff
@@ -0,0 +1,28 @@
+---
+to: sr2020-teams
+subject: Confirming Kickstart details
+---
+
+Hi all!
+
+Kickstart is only just around the corner, so it's time to confirm details for the kickstart events happening this year.
```

Comment:
[nit] this sounds like the teams are meant to confirm the details (maybe this should be 'so we're happy to confirm the details'...)

Reply:
ooh, good catch!